lib/compiler.js

```diff
@@ -598,3 +604,69 @@ function resolveAppScriptPath(rootDir, relativePath, resolveOptions) {
   var fixedFile = fixFileExtension(resolvedPath, files, false);
   return (fixedFile === undefined ? resolvedPath : fixedFile);
 }
+
+function buildAllMixinInstructions(appRootDir, mixinDirs, options) {
+  var extensions = _.without(_.keys(require.extensions),
+    _.keys(getExcludedExtensions()));
+  var files = options.mixins || [];
+
+  mixinDirs.forEach(function(dir) {
+    dir = tryResolveAppPath(appRootDir, dir);
```

Comment:
Please use the strict mode here, so that `mixins` is always resolved via `node_modules` and never as `./mixins`. 

I think it will be best to make the `strict` option enabled by default, and explicitly set `strict:false` only in the few places where we need to preserve backwards compatibility.

Thoughts?